src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Order.xml

```diff
@@ -306,6 +306,17 @@
                     <attribute name="summary">Applies coupon to cart</attribute>
                 </attribute>
             </itemOperation>
+
+            <itemOperation name="shop_put">
+                <attribute name="method">PUT</attribute>
+                <attribute name="path">/shop/orders/{id}</attribute>
+                <attribute name="denormalization_context">
+                    <attribute name="groups">cart:update</attribute>
+                </attribute>
+                <attribute name="openapi_context">
+                    <attribute name="summary">Change locale code on cart</attribute>
+                </attribute>
```

Comment:
IMHO it is redundant, we don't need to explain normal `put` endpoint

Reply:
But now it's only available parameter on order put method which you can change.

Reply:
still, it is normal `put`, on, for example, product `put` will you list all available parameter?